samples/python/tutorial_code/features2D/Homography/panorama_stitching_rotating_camera.py

```diff
@@ -0,0 +1,66 @@
+import cv2 as cv
+import numpy as np
```

Comment:
`numpy` should go first (as dependency of opencv - we have reports about that):

Use this header:
```
#!/usr/bin/env python
# -*- coding: utf-8 -*-

# Python 2/3 compatibility
from __future__ import print_function

import numpy as np
import cv2 as cv
```